GrandCentralBoard/Widgets/Image/ImageWidget.swift

```diff
@@ -20,7 +20,11 @@ final class ImageWidget: WidgetControlling {
     }
 
     var view: UIView {
-        return widgetView
+        guard isHeaderVisible else { return widgetView }
+
+        let viewModel = WidgetTemplateViewModel(title: "CAT PHOTOS", subtitle: "NEWEST CAT PROFILES", contentView: widgetView)
+        let layoutSettings = WidgetTemplateLayoutSettings(contentMargin: UIEdgeInsetsZero, displayContentUnderHeader: true)
+        return WidgetTemplateView.viewWithViewModel(viewModel, layoutSettings: layoutSettings)
```

Comment:
Similar issue to https://github.com/macoscope/GrandCentralBoard/pull/140#discussion_r63876301. Let's construct this wrapper view somewhere else.